Gradle 6.4 problems

Upgrading gradle to 6.4 causes build problems.

```
> Task :liquibaseUpdate FAILED                                                                                                                                                                                                                                
Caching disabled for task ':liquibaseUpdate' because:                                                                                                                                                                                                         
  Build cache is disabled                                                                                                      
Task ':liquibaseUpdate' is not up-to-date because:                                                                             
  Task has not declared any outputs despite executing actions.                                                                 
Excluding []                                                                                                                   
:liquibaseUpdate (Thread[Execution worker for ':' Thread 9,5,main]) completed. Took 0.033 secs.                                                                                                                                                               
                                                                                                                                                                                                                                                              
FAILURE: Build failed with an exception.                                                                                                                                                                                                                      
                                                                                                                               
* What went wrong:                                                                                                             
Execution failed for task ':liquibaseUpdate'.                                                                                  
> The value for task ':liquibaseUpdate' property 'mainClass' is final and cannot be changed any further.                
                                                                                                                                                                                                                                                              
* Try:                                                                                                                                                                                                                                                        
Run with --debug option to get more log output. Run with --scan to get full insights.                                        
                                                                                                                                                                                                                                                              
* Exception is:                                                                                                                                                                                                                                               
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':liquibaseUpdate'.                                                                                                                                                                    
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:207)                                                                                                                    
        at org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:263)                                       
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:205)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:186)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:114)
        at org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)
        at org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57) 
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)
```

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/src/main/groovy/org/liquibase/gradle/LiquibaseTask.groovy
+++ b/src/main/groovy/org/liquibase/gradle/LiquibaseTask.groovy
@@ -17,6 +17,7 @@
 
 package org.liquibase.gradle
 
+import org.gradle.api.Task
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.JavaExec
 import org.gradle.api.tasks.TaskAction
@@ -137,11 +138,18 @@ class LiquibaseTask extends JavaExec {
 //		configureLogging(classpath, project.liquibase.mainClassName)
 //		setSystemProperties(["mainClass": project.liquibase.mainClassName])
 //		setMain("org.liquibase.gradle.LiquibaseRunner")
-		setMain(project.liquibase.mainClassName)
 		// "inherit" the system properties from the Gradle JVM.
 		systemProperties System.properties
 		println "liquibase-plugin: Running the '${activity.name}' activity..."
 		project.logger.debug("liquibase-plugin: Running 'liquibase ${args.join(" ")}'")
 		super.exec()
+	}
+
+	@Override
+	Task configure(Closure closure) {
+		conventionMapping("main") {
+			project.extensions.findByType(LiquibaseExtension.class).mainClassName
+		}
+		return super.configure(closure)
 	}
 }

